src/app/[locale]/(main)/profile/[id]/account/actions/change-password.tsx

```diff
@@ -0,0 +1,46 @@
+import { getAPIServerURL } from "@/lib/utils"
+
+type ChangePasswordSchema = {
+  token: string
+  values: {
+    oldPassword: string
+    password: string
+  }
+}
+
+export const ChangePassword = async ({
```

Comment:
all action server must be mark as "use server" and function have "action" as suffix (ex: getUserProfileAction())